js/bootstrap-datepicker.js

```diff
@@ -1543,6 +1543,10 @@
 				p.setRange(range);
 			});
 		},
+		clearDates: function(){
+				this.pickers[0].clearDates();
+				this.pickers[1].clearDates();
```

Comment:
I'm not sure if we should check for the 0 and 1 key or just loop over all `this.pickers`. In theory there should only be 2 pickers, what do you think @coderofsalvation 

/cc @Azaret @vsn4ik